node/config/load.go

```diff
@@ -395,8 +398,32 @@ func ConfigUpdate(cfgCur, cfgDef interface{}, opts ...UpdateCfgOpt) ([]byte, err
 			return nil, xerrors.Errorf("parsing updated config: %w", err)
 		}
 
-		if !reflect.DeepEqual(cfgCur, cfgUpdated) {
-			return nil, xerrors.Errorf("updated config didn't match current config:")
+		opts := []cmp.Option{
+			// This equality function compares big.Int
+			cmpopts.IgnoreUnexported(big.Int{}),
+			cmp.Comparer(func(x, y []string) bool {
+				//both nil
+				if x == nil && y == nil {
+					return true
+				}
+				// both []string
+				if reflect.TypeOf(x) == reflect.TypeOf(y) {
+					return true
+				}
+				// one of them is nil
+				if reflect.TypeOf(x) == nil && reflect.TypeOf(x) != reflect.TypeOf(y) {
```

Comment:
reflect.TypeOf( anything) != nil 
so these 2 cases (Line 414 & 418) can never happen

Reply:
My bad! Nil is just a Pointer. Not sure what I was thinking there.